samples/CommunityToolkit.Maui.Sample/CommunityToolkit.Maui.Sample.csproj

```
@@ -42,14 +42,15 @@
 
     <!-- Images -->
     <MauiImage Include="Resources\Images\*" />
+    <MauiImage Update="Resources\Splash\spash.svg" BaseSize="128,128" />
```

Should we be using `<MauiSplashScreen Include= ... />` instead of `<MauiImage Update=... />`?

```suggestion
    < MauiSplashScreen Include="Resources\Splash\spash.svg" BaseSize="128,128" />
```

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/splashscreen?tabs=ios

---

Let me try it out.  I used this as my guide on how to resolve, which used MauiImage Update.
https://learn.microsoft.com/en-us/answers/questions/1178661/how-to-fix-error-dep0700-maui-net7-0

---

@brminnick That's the issue.  On the Windows platform, the spash screen icon is not found when using just MauiSplashScreen.  See the link I provided to the .NET MAUI bug (@jfversluis was involved in the bug).

---

@brminnick @VladislavAntonyuk  Typo fixed and build fixed on the Windows platform.